### PR TITLE
adding house 43

### PR DESCRIPTION
**Pull request by LilShiningMan**
#### Summary
```SUMMARY: Content "adding new residence - house 43"```

#### Purpose of change
expanding variety in residential building spawns

#### Describe the solution
added new house with 3 basement variants and 4 ground level variants, including one rare "on fire" variant and one rare "crashed vehicle" variant

#### Describe alternatives you've considered
not adding new houses.

#### Testing
Linted and spawned house in game. 

#### Additional context
second house I've added, many more to come.

house33_ground1
house33_ground2
house33_ground3
house33_ground4
house33_basement1
house33_basement2
house33_basement3
house33_roof

### CODE REVIEW

**Comment by curstwist** [comment]
Could you please rename house 43?  Fuji's structures uses 33-42 and will be moved to mainline when I have time to add them.

https://github.com/CleverRaven/Cataclysm-DDA/tree/master/data/mods/Fuji_Structures/worldgen/houses

File: data/json/mapgen/house/house33.json [hunk 1]
```diff
@@ -0,0 +1,631 @@
...
+        "R": "t_carpet_yellow",
+        "]": "t_carpet_yellow",
+        "T": "t_carpet_yellow"
+      },
+      "furniture": {
+        "G": "f_deckchair",
+        "%": "f_makeshift_bed",
+        "&": "f_table",
+        "k": "f_chair",
+        "0": [
```
Comment (curstwist) [inline data/json/mapgen/house/house33.json]: `"f_region_flower"` is a good option here to condense the mapgen.

File: data/json/mapgen/house/house33.json [hunk 2]
```diff
@@ -0,0 +1,631 @@
...
+        "}........=....|?????|..}",
+        "}..=...........,,,,,/..}",
+        "}}}}}}}}}}}}p..,,,,,...}"
+      ],
+      "palettes": [ "standard_domestic_palette", "standard_domestic_lino_bathroom", "standard_domestic_lino_kitchen" ],
+      "terrain": {
+        "?": "t_door_metal_locked",
+        ",": "t_pavement",
+        "/": "t_gates_mech_control",
+        "=": [
```
Comment (curstwist) [inline data/json/mapgen/house/house33.json]: [ [ "t_region_tree_fruit", 2 ], [ "t_region_tree_nut", 2 ], "t_region_tree_shade" ] (any or all of these regional options for trees).

File: data/json/overmap/multitile_city_buildings.json [hunk 3]
```diff
@@ -510,6 +510,16 @@
...
     ]
   },
+  {
+    "type": "city_building",
+    "id": "house_33",
+    "locations": [ "land" ],
+    "overmaps": [
+      { "point": [ 0, 0, -1 ], "overmap": "house_33_basement_north", "locations": [ "land" ] },
+      { "point": [ 0, 0, 0 ], "overmap": "house_33_north", "locations": [ "land" ] },
+      { "point": [ 0, 0, 1 ], "overmap": "house_33_roof_north", "locations": [ "land" ] }
```
Comment (curstwist) [inline data/json/overmap/multitile_city_buildings.json]: remove the "locations" entry in these lines.

**Comment by curstwist** [comment]
overall looks great.  a few notes:
garage wench doesn't work.  you want something like:
image
see [house_garage2](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/mapgen/house/house_garage2.json#L11) for an example.

besides, that.. add another gutter downspout, houses get 2 usually.


You might want to start exploring nested mapgen too if you plan on doing a lot of houses.  Looking forward to seeing more : )

**Comment by curstwist** [comment]
I had time to review these closer.

field effects:
The variation (Ground Floor - Variant 4: fire) that has the bed fire should be adjusted to remove the fire.  We don't want spontaneous fires in cities since it could ignite the entire block.  This needs more discussion if you want to begin adding environmental dangers to city spawns.

rotation:
Since you're planning on adding more houses, the best practice is to orient your rows so the road edge is at line 0 (north ward).  For this one, you've oriented it for a southern road and added `"rotation": 2` instead.  Almost all of our maps are north oriented, so we'd like to maintain consistency.

**Comment by LilShiningMan** [comment]
In the case of the house fire variant, that's exactly the kind of thing I want to have happen. For a game about the end of the world due to alien invasion/zombie apocalypse, it sometimes feels like things are a little too neat and static, and having the potential for extra chaos is desirable in my opinion. However, I recognize this is just my opinion, and not everyone wants potential block-wide fires, and therefore this probably belongs more in an opt-in mod.

As for the house orientation, that's simply my preference when working, and it's functionally indistinguishable for players. I understand your point about uniformity of file formats, but this would require time spent converting maps and making changes to how I work. At the moment I'd rather spend that time creating content. 

In the future I may submit more content PRs, but in the meantime I think I'll delete this PR and consider my options about how to proceed with the maps I've already created. 

As an aside, I had a house merged last week (2storymodern01) that also has a similar orientation. If that is similarly judged to be not fitting in with the rest of the maps I can "take it back" and add it to whatever mod or content pack I put together. 

Thanks for the review and comments!

**Comment by curstwist** [comment]
The orientation style wouldn't keep this from getting merged.  It's a request help keep the maps consistent for future contributors.

The fields additions require a larger discussion instead of automatically being added through house mapgen PR since it has larger game play impacts.  The development discord is a good place to have this discussion or in a Github issue.

**Pull request closed without merging**